tfhe/src/integer/ciphertext/mod.rs

```diff
@@ -263,20 +270,18 @@ impl IntegerRadixCiphertext for SignedRadixCiphertext {
 }
 
 impl IntegerCiphertext for CrtCiphertext {
+    fn blocks(&self) -> &[Ciphertext] {
+        &self.blocks
+    }
+
     fn from_blocks(blocks: Vec<Ciphertext>) -> Self {
         let moduli = blocks.iter().map(|x| x.message_modulus.0 as u64).collect();
         Self { blocks, moduli }
     }
-    fn blocks(&self) -> &[Ciphertext] {
-        &self.blocks
-    }
+
     fn blocks_mut(&mut self) -> &mut [Ciphertext] {
         &mut self.blocks
     }
-
-    fn moduli(&self) -> Vec<u64> {
-        self.moduli.clone()
-    }
```

Comment:
never used ?

Reply:
It was removed by mistake, I will re-add it, removing it makes it use the default impl of the trait which is still correct, but a bit less optimal